src/main.rs

```diff
@@ -98,9 +98,8 @@ fn main() {
 
     let mut previous_total_processed: Option<u32> = None;
 
+    debug!("Welcome to datadog-sidekiq");
```

Comment:
given the usage of `info!` around the existing code base I could see this as `info!` as well (rather than `debug!`).